genisys/server/__init__.py

```diff
@@ -11,6 +11,7 @@
 from genisys.server.genisysinventory import GenisysInventory
 from genisys.server.http import GenisysHTTPServer, GenisysHTTPRequestHandler
 import genisys.server.tls
+import dotenv
```

Comment:
Make sure this is in the dependencies. Looks like this is the main reason the tests are currently failing. Other than that looks great.

Reply:
Ok, I will add this when I can

Reply:
done